src/com/lftechnology/assignment/jan7/FileInput.java

```diff
@@ -0,0 +1,145 @@
+package com.lftechnology.assignment.jan7;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+/**
+ * 5. Write a program called FileInput to read an int, a double, and a String form a text file called "in.txt", and produce the following
+ * output: (Hint use Scanner to read from file) a. The integer read is 12 b. The floating point number read is 33.44 c. The String read is
+ * "Peter" d. Hi! Peter, the sum of 12 and 33.44 is 45.44
+ * 
+ * This class takes in input provide in a text file and print the result in the specific format.
+ * 
+ * @author bibhushan<bibhushanjoshi@lftechnology.com>
+ * 
+ */
+public class FileInput {
+	private static final Logger LOGGER = Logger.getLogger(FileInput.class.getName());
+
+	public static void main(String[] args) {
+		Scanner scanUserInput = null;
+		createOutputMessage(scanUserInput);
+	}
+
+	/**
+	 * This method appends the string to create an output message.
```

Comment:
I cannot understand what this method does. Should be more clear. Sometimes, it is better to write example as well if needed.